mk/spksrc.service.installer

```diff
@@ -20,6 +23,21 @@ else
     TMP_DIR="${SYNOPKG_TEMP_UPGRADE_FOLDER}/var"
 fi
 
+
+install_log ()
+{
+  local _msg_="$@"
+  if [ -z "${_msg_}" ]; then
+      # read multiline from stdin
+      while IFS=$'\n' read -r line; do
+        install_log "${line}"
+      done
+  else
+    echo $(date +'%Y/%m/%d %H:%M:%S')$'\t'${_msg_} >> ${INST_LOG} 2>&1
```

Comment:
```suggestion
    echo $(date +'%Y/%m/%d %H:%M:%S')$'\t'${_msg_} >> ${INST_LOG} 1>&2
```

Reply:
in the pre dsm7 installer, we want to redirect everyting (stdout and stderr) to the log file (not to stderr).